test/functional/mempool_limit.py

```diff
@@ -85,6 +85,99 @@ def test_rbf_carveout_disallowed(self):
+        # (note that the mempool usage of a tx is about three times its vsize)
+        assert_greater_than(parent_vsize * num_big_parents * 3, current_info["maxmempool"] - current_info["bytes"])
+
+        big_parent_txids = []
+        big_parent_wtxids = []
+        for i in range(num_big_parents):
+            # Last parent is higher feerate causing other parents to be possibly
+            # be evicted if trimming was allowed, which would cause the package to end up failing
+            parent_feerate = mempoolmin_feerate + Decimal("0.00000001") if i == num_big_parents - 1 else mempoolmin_feerate
+            parent = self.wallet.create_self_transfer(fee_rate=parent_feerate, target_vsize=parent_vsize, confirmed_only=True)
+            parent_utxos.append(parent["new_utxo"])
+            package_hex.append(parent["hex"])
+            big_parent_txids.append(parent["txid"])
+            big_parent_wtxids.append(parent["wtxid"])
+            # There is room for each of these transactions independently
+            assert node.testmempoolaccept([parent["hex"]])[0]["allowed"]
+
+        # Create a child spending everything with an insane fee, bumping the package above mempool_entry_minrate
+        child = self.wallet.create_self_transfer_multi(utxos_to_spend=parent_utxos, fee_per_output=10000000)
+        package_hex.append(child["hex"])
+
+        # Package should be submitted, temporarily exceeding maxmempool, but not evicted.
+        package_res = None
+        with node.assert_debug_log(expected_msgs=["rolling minimum fee bumped"]):
+            package_res = node.submitpackage(package=package_hex, maxfeerate=0)
+
+        assert_equal(package_res["package_msg"], "success")
+
+        # If intra-packing trimming ends up happening anyways, we would
+        # see this show up in effective-includes results when a parent
+        # and child package are evaluated together as a single subpackage
+        assert_equal(len(package_res["tx-results"]), len(big_parent_wtxids) + 1)
+        for wtxid in big_parent_wtxids + [child["wtxid"]]:
+            assert_equal(len(package_res["tx-results"][wtxid]["fees"]["effective-includes"]), 1)
+
+        # Maximum size must never be exceeded.
+        assert_greater_than(node.getmempoolinfo()["maxmempool"], node.getmempoolinfo()["bytes"])
+
+        # Package found in mempool still
+        resulting_mempool_txids = node.getrawmempool()
+        assert child["txid"] in resulting_mempool_txids
+        for txid in big_parent_txids:
+            assert txid in resulting_mempool_txids
+
+        # Check every evicted tx was higher feerate than parents which evicted it
+        eviction_set = set(mempool_txids) - set(resulting_mempool_txids) - set(big_parent_txids)
+        parent_entries = [node.getmempoolentry(entry) for entry in big_parent_txids]
+        max_parent_feerate = max([entry["fees"]["base"] / (Decimal(entry["vsize"]) / 1000) for entry in parent_entries])
```

Comment:
nit: strictly speaking...
```suggestion
        max_parent_feerate = max([entry["fees"]["modified"] / (Decimal(entry["vsize"]) / 1000) for entry in parent_entries])
```

Reply:
changed in both places